### DISCUSSION TITLE
Exporting as .epub

### DISCUSSION THREAD

**Discussion by YourTheFinestIveEverKnown** [Zettlr/Zettlr#4763] [Q&A]
I need to export my projects (books) from Zettlr as .epub, but I do no see this explicit format in the exporting options. Advice? I have been reading the Zettlr documentation and linked content about Textbundle/Textpack/Simple PDF, but I don't know how these best relate to .epub. Is there perhaps a plugin available I can export directly to .epub from Zettlr? I really don't know.

**Comment by lordofscripts** [comment]
After I solve the issue with specifying the logical (rather than alphabetical) order of the files, this export format would be the next great thing!

**Reply by YourTheFinestIveEverKnown** [reply]
It really would be. Thanks so much!
